cluster/ubuntu/util.sh

```diff
@@ -450,7 +450,61 @@ function kube-down {
 
 # Update a kubernetes cluster with latest source
 function kube-push {
-  echo "not implemented"
+  #stop all the kube's process & etcd 
+  KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
```

Comment:
KUBE_ROOT should already be set here.

https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-push.sh#L26